docs/de_DE/settings.md

```diff
@@ -8,7 +8,7 @@ lang: de_DE
 In den Einstellungen werden die Einstellungen aller Module zentral verwaltet. Es können nur die Einstellungen aktiver Module verändert werden.
 
 Mittels der Knöpfe `Export` und `Import` kannst du deine Einstellungen in einer Datei sichern und / oder an deine Freunde weitergeben.
-Wir möchten in naher Zukunft die Möglichkeit bieten, Einstellungen Profilgebunden zu speichern. Damit sind Einstellungen nichtmehr Gerätegebunden.
+Wir möchten in naher Zukunft die Möglichkeit bieten, Einstellungen Profilgebunden zu speichern. Damit sind Einstellungen nicht mehr Gerätegebunden.
```

Comment:
gerätegebunden wäre grammatikalisch korrekt. Würde das aber umformulieren in "an das Gerät gebunden" bzw. "an ein Gerät gebunden" oä

Reply:
=> d6e346d